shell/platform/linux/BUILD.gn

```diff
@@ -217,6 +217,12 @@ shared_library("flutter_linux_gtk") {
   deps = [ ":flutter_linux" ]
 
   public_configs = [ "//flutter:config" ]
+
+  version_script =
```

Comment:
Maybe also add this to `shared_library("flutter_linux_glfw")` cc @stuartmorgan.

Reply:
That library isn't supported for general use at this point, but it seems reasonable to add it to both.

There's also `flutter_engine.so`, which is used by custom embedders; presumably that should get essentially the same fix. /cc @chinmaygarde

Reply:
This seems like an odd place to fix symbols using incorrect visibility. IMO, depending on the `//third_party/cxxabi` target should not make the users also have to manually restrict visibility of symbols in that target. We should be able to set `_LIBCXXABI_DISABLE_VISIBILITY_ANNOTATIONS` in the the public GN configs of `//third_party/cxxabi`. That'll suppress any leaks. Can we file a bug for this (with a work item to add a symbol check to `flutter_engine.so` as well)?